src/textual_dev/client.py

```diff
@@ -156,9 +159,18 @@ async def send_queued_logs():
                     await websocket.send_bytes(log)
                 log_queue.task_done()
 
+        async def server_info_received() -> None:
+            """Wait for the first server info message to be received and handled."""
+            try:
+                await asyncio.wait_for(self._ready_event.wait(), timeout=READY_TIMEOUT)
+            except asyncio.TimeoutError:
+                return
```

Comment:
If there's a time out, what's happening is that we just give up on waiting for the server info?

Reply:
Yes. We'll wait for the server info for a maximum of READY_TIMEOUT. If the server info doesn't arrive, it's not fatal, so continue.

It's nice to wait for it rather than continuing without it because otherwise we'll process log messages before we've received the console size info and things don't wrap correctly.

Reply:
Alright, then I understood it correctly 👍